homeassistant/components/rachio/__init__.py

```diff
@@ -285,15 +356,18 @@ class RachioWebhookView(HomeAssistantView):
     url = WEBHOOK_PATH
```

Comment:
This should be rewritten to use our webhooks implementation in a future PR.

Reply:
Actually, having this fixed url being bound to a single entry means that right now you are still limited to 1 rachio account until it starts using our built-in webhooks

Reply:
I'm thinking the interim step is to append the config entry_id to the url and then switch this to the way netatmo does it in a new PR (if that is the best example)

Reply:
I added the entry_id to the URL so they have a unique one.  I used two different accounts to test with and ensured that the live updates came back to the separate endpoints.

Reply:
Converted to cloud hooks here
https://github.com/home-assistant/core/pull/33724